_test/test_sqw/test_mask.m

```diff
@@ -259,13 +259,13 @@ function test_num_pix_has_been_reduced_by_correct_amount_paged_pix_3d(obj)
         function test_img_db_range_recalculated_after_mask_3d(obj)
             original_img_db_range = obj.sqw_3d.data.img_db_range;
             img_db_range_diff = abs(original_img_db_range - obj.masked_3d.data.img_db_range);
-            assertTrue(~obj.fh_range_check(img_db_range_diff,0.001));
+            assertTrue(obj.fh_range_check(img_db_range_diff,1.e-7));
```

Comment:
Tightening these tolerances?

Reply:
Might want to switch these to the "official" [1.e-7, 0]

Reply:
The behavior have changed. Masking does not change img_db_ranges as this is the range where pixels were rebinned into, so masking would not change it unless you rebin. 
 `fh_range_check` does not recognizes [1.e-7,0].

Reply:
Sorry, I thought it would use `equal_to_tol` underneath.